packages/vats/src/lib-board.js

```diff
@@ -32,40 +31,50 @@ const calcCrc = num => {
 /**
  * Create a board to post things on.
  *
- * @param {number} [seed=0]
+ * @param {number | bigint} [firstSequence=0]
+ * @param {string} [prefix]
  * @returns {Board}
  */
-function makeBoard(seed = 0) {
+function makeBoard(firstSequence = 0, prefix = 'board0') {
   const idToVal = makeStore('boardId');
   const valToId = makeStore('value');
-  const sparseInts = generateSparseInts(seed);
+  let lastSequence = BigInt(firstSequence);
 
   /** @type {Board} */
   const board = Far('Board', {
     // Add if not already present
     getId: value => {
       if (!valToId.has(value)) {
-        // Retry until we have a unique id.
-        let id;
-        do {
-          const num = /** @type {number} */ (sparseInts.next().value);
-          id = `${num}${calcCrc(num)}`;
-        } while (idToVal.has(id));
+        lastSequence += 1n;
+        const seq = `${lastSequence}`;
+
+        // Reverse the sequence, so that the initial part of the board ID is
+        // well-distributed.
+        const reverseSeq = seq.split('').reverse().join('');
+
+        // Append the CRC, so that the last part of the board ID is
+        // well-distributed.
```

Comment:
If it's important to be well-distributed, would it suffice to have `board0${crc}${serial}`? Somewhat simpler to express and parse.

Reply:
It's a tradeoff, but I can agree with your suggestion.  Your comment addresses the constraint I was trying to express.